sass/_vars.scss

```diff
@@ -94,12 +94,12 @@ $color-white: #fff;
   --color-white: #{$color-white};
 
   // Blues by lightness
-  --color-blue-21: #2f2745;
+  --color-blue-21: #332e41;
```

Comment:
Does it make sense to change these definitions directly? It appears to be changing the lightness, which is what the numbers represent